src/lib/openjp2/openjpeg.c

```diff
@@ -33,7 +33,30 @@
 
 #ifdef _WIN32
 #include <windows.h>
-#endif /* _WIN32 */
+#include <fcntl.h>
+
+#else /* _WIN32 */
+
+#include <errno.h>
+
+#include <stdarg.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+
+#ifdef HAVE_UNISTD_H
```

Comment:
The test for this macro must be enabled in src/lib/openjp2/opj_config_private.h.cmake.in.